llvm/test/CodeGen/AMDGPU/preload-implicit-kernargs-IR-lowering.ll

```diff
@@ -83,6 +88,8 @@ define amdgpu_kernel void @preloadremainder_z(ptr addrspace(1) %out) {
   ret void
 }
 
+; PRELOAD-NOT: declare {{.*}}@1
```

Comment:
better to use -implicit-check-not=declare

Reply:
This doesn't work because of implicit_arg ptr declaration.

Reply:
Can you relax it to `@{{[0-9]+}}`?